frontend/plugins/pos_ui/src/modules/create-pos/components/sync/sync.tsx

```
@@ -71,6 +78,26 @@ export default function SyncCardForm({
     });
   };
 
+  const handleBranchChange = (branchId: string | string[] | undefined) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'handleBranchChange'. Consider to delete it or rename it.

'handleBranchChange' is defined here:


(lint/suspicious/noRedeclare)